backend/src/controllers/question.js

```diff
@@ -14,12 +14,11 @@ exports.getQuestion = function ({ user, params }, res) {
         likesModel.getLikes({ type: 'question', id: params.id }),
         likesModel.getUserLiked({ type: 'question', id: params.id, userID })
     ]).then(([question, likes, userLiked]) => {
-        return Promise.all([
-            userModel.getPublicProfile(question.userID)
-        ]).then(([userInfo]) => {
-            delete question.userID
-            return { ...question, ...likes, ...userLiked, user: userInfo }
-        })
+        return userModel.getPublicProfile(question.userID)
```

Comment:
This doesn't need to be nesting deeper and the return is redundant if u take out the curly bracket.